docs/docs/guides/token_expiration.mdx

```diff
@@ -0,0 +1,103 @@
+---
+id: token_expiration
+title: Configuring Access Token Expiration Time
```

Comment:
I think we should make this more generic given that this document explains more than just one expiry time:

```suggestion
title: Configuring Expiration
```